WINDOWS.md

```diff
@@ -77,9 +77,47 @@ deactivate
 cp config-example.yaml config.yaml
 ```
 
-6. Open up the newly created `config.yaml` in your favorite text editor and configure it 
+6. Open up the newly created `config.yaml` in your favorite text editor and configure it
    to your preferences.
 
+## 1.2 WSL(Ubuntu) Installation
+
+Reminder: `INFO` level log in Chia must be enabled (`chia configure -log-level=INFO`)
+
+1. Open `WSL-Ubuntu`
```

Comment:
I would explain briefly what WSL is and provide a link to an article detailing its installation.